charts/datadog/templates/_helm_check_config.yaml

```diff
@@ -0,0 +1,9 @@
+{{- define "helmCheck-config" -}}
+helm.yaml: |-
+{{- if .Values.datadog.clusterChecks.enabled }}
+  cluster_check: true
+{{- end }}
+  init_config:
+  instances:
```

Comment:
doest the "leader" option is not need

Reply:
I don't think so: https://github.com/DataDog/datadog-agent/pull/10949#discussion_r807058890
But maybe I'm missing something.